src/components/ha-icon-next.ts

```diff
@@ -0,0 +1,25 @@
+import "@polymer/iron-icon/iron-icon";
+// Not duplicate, this is for typing.
+// tslint:disable-next-line
+import { HaIcon } from "./ha-icon";
+
+export class HaIconNext extends HaIcon {
+  public connectedCallback() {
+    super.connectedCallback();
+
+    this.icon =
+      window.getComputedStyle(this).direction === "ltr"
+        ? "hass:chevron-right"
+        : "hass:chevron-left";
+
+    this.fire("iron-resize");
```

Comment:
why this?

Reply:
Well because until I added this reloading the page would sometimes not show the icon and I saw that in many places in the code this was done even though no real "resize" was happening so I assumed this is a refresh of some sort and indeed it got the work done. Any other suggestions welcome.

Reply:
It's being used when DOM elements change size or are shown/hidden, like images being loaded or tab bars hidden/shown. We shouldn't just hack our way into this here because it works.

Reply:
Fixed. Calling super after setting the icon fixes it.